src/main/java/core/basesyntax/service/WriteToFileService.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteToFileService {
```

Comment:
```suggestion
public class WriteToFileServiceImpl implements WriteToFileService {
```
add abstraction for all services